src/Backend/Backend.ts

```diff
@@ -39,7 +41,7 @@ function backendRegistrationApi() {
       if (compiler) {
         gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
       }
-      console.log(`Backend ${backendName} was registered into ONE-vscode.`);
+      Logger.info(logTag, 'Backend', backendName, 'was registered into ONE-vscode.');
```

Comment:
![image](https://user-images.githubusercontent.com/63768207/170210050-73089725-d9a1-41c2-acd3-5e04678d64e6.png)

Reply:
type of `backendName` is string.

Reply:
Note: Previous code actually does not need to be changed to this code.... but I changed just to make a sample where `string` can be passed as one of params.
Previous code (one single msg string) with `Logger.info` like
`Looger.info(tag, `Backend ${backendName} was registered into ONE-vscode.`);` also works.